### ISSUE TITLE
Markdown in email - value.includes in escapeQuotes

### ISSUE THREAD

**Issue by MartinN3** [resend/react-email#1632]
### Describe the Bug

When using markdownCustomStyles, parseCssInJsToInlineCss calls escapeQuotes on `fontWeight: 500,` in h1 tag and that fails but when using `fontWeight: '500'` it goes trough.

### Which package is affected (leave empty if unsure)

react-email

### Link to the code that reproduces this issue

is there a default repo we can clone and tailor?

### To Reproduce
```
<Markdown
        children={`# Welcome to`}
        markdownCustomStyles={{
          h1: {
            fontSize: 70,
            fontWeight: 500,
          },
        }}
      />
 ```
      
```javascript
"dependencies": {
        "@react-email/components": "0.0.23",
        "@react-email/render": "^1.0.0",
        "express": "4.19.2",
        "react": "18.3.1",
        "react-dom": "18.3.1",
        "react-email": "^3.0.1",
        "typescript": "^5.5.4"
 },
 ```

### Expected Behavior

I think you would wanna override h1 fontWeight and you would do it easily by specifying fontWeight: 500, right?

**Comment by VedantPatel04** [comment]
I'm working on this!

**Comment by Hainesy** [comment]
Any update on this? Thanks

**Comment by gabrielmfern** [comment]
Just released `@react-email/markdown@0.0.13-canary.1`/`@react-email/components@0.0.29-canary.2` that fixes this issue. Once we merge the fix into stable, I'll close out this issue.
